src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx

```diff
@@ -230,39 +255,43 @@ const ManageItem: FC<{
   setManage: (flag: boolean) => void;
   action?: () => void;
   isExplorer?: boolean;
-}> = ({ title, id, setManage, action, isExplorer }) => {
+  disabled?: boolean;
+}> = ({ title, id, setManage, action, isExplorer, disabled }) => {
   const tooltipText = getTooltipTextByStrategyEditOptionsId(isExplorer)?.[id];
   const { belowBreakpoint } = useBreakpoints();
 
+  const Content = () => {
+    return (
+      <div
+        onClick={() => {
+          if (disabled) return;
+
+          action && action();
+          setManage(false);
+        }}
+        className={cn('rounded-6 p-12', {
+          'cursor-pointer': !disabled,
+          'cursor-not-allowed': disabled,
```

Comment:
Common behavior for disabled button is usually "pointer-events: none", like that you don't need to verify `if (disabled)` above